app/src/db/migrations/versions/2023_08_10_default_table_privileges.py

```diff
@@ -0,0 +1,24 @@
+"""default table privileges
+
+Revision ID: 3ed861176e3d
+Revises:
+Create Date: 2023-08-10 15:52:10.626153
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "3ed861176e3d"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Change default privileges for future tables created by the `migrator`
+    # user to automatically be accessible by the `app` user.
+    op.execute("ALTER DEFAULT PRIVILEGES GRANT ALL ON TABLES TO app")
```

Comment:
Do we want to add more rules than just tables?

When I set this up for local development on my project, I did this for each schema, borrowing some of it from our PFML project:
- `REVOKE ALL ON SCHEMA PUBLIC FROM PUBLIC"`
- `GRANT USAGE ON SCHEMA {schema_name} TO {role};` (I think this is automatic for the public schema)
- `ALTER DEFAULT PRIVILEGES IN SCHEMA {schema_name} GRANT ALL ON TABLES TO {role};`
- `ALTER DEFAULT PRIVILEGES IN SCHEMA {schema_name} GRANT ALL ON SEQUENCES TO {role};`
- `ALTER DEFAULT PRIVILEGES IN SCHEMA {schema_name} GRANT ALL ON ROUTINES TO {role};`

Sequences and routines are a bit less necessary, but threw them in

Reply:
Yeah those are all great. The idea I had was to do as much as possible in the infra template, so the first two items (1) `REVOKE ALL ON SCHEMA public FROM public` and (2) `GRANT USAGE ON SCHEMA {schema_name} TO {role}` are going to be done in template-infra. (2) is already done and (1) is captured by [this ticket](https://github.com/navapbc/template-infra/issues/300).

I'm fine adding SEQUENCES and ROUTINES to this PR.

Reply:
Ok added

Reply:
Hmm - just thinking of a developer in the future looking at this (or at the piece in the infra), I'd wonder why those two are split into two separate places. Could you add a comment here explaining that? And maybe add to that other ticket to update the comment to link to that bit of code so a developer can see the full picture?

Reply:
Good point, added a comment and updated the ticket.

Reply:
@chouinar no rush but lemme know if you're good with the latest changes